controllers/front/ChangeCurrencyController.php

```diff
@@ -36,7 +36,11 @@ public function initContent()
         if (Validate::isLoadedObject($currency) && !$currency->deleted) {
             $this->context->cookie->id_currency = (int)$currency->id;
             $this->ajaxRender('1');
+
+            return;
         }
         $this->ajaxRender('0');
```

Comment:
Could be `return $this->ajaxRender('0'); ` if you prevent someone from adding code :) Same for every `return;`, just return the latest function called?

Reply:
yes but Phpstorm will complain you return from a function returning void :)